### ISSUE TITLE
Review of molecule dynamic description

### ISSUE THREAD

**Issue by terracoda** [phetsims/molecules-and-light#323]
@Matthew-Moore240, based on the changes we made in the [Table: Review of Molecule Responses](https://docs.google.com/document/d/1r9D3vq4QvmmXkQ5G6HexpybUv_vL6LrYFb3wgPfUdWg/edit#heading=h.8gkq5oll4829), can you review the language we used in the dynamic description in the Observation Window in Play Area.

We want to be using the same language as much as possible. It looks like the molecule dynamic description may be sharing the same parameters.

Check with @jessegreenberg, it might be easier to review once the work in #285 is complete.

**Comment by jessegreenberg** [comment]
@Matthew-Moore240 gave me this spreadsheet of updated dynamic descriptions for the first sentence of the dynamic observation window description: https://docs.google.com/spreadsheets/d/19eCpnjjaJ8it0Tzdbk_159xhH_WJVPVmv_jHwnSdw3s/edit#gid=0

Assigning to myself to implement.

**Comment by jessegreenberg** [comment]
If I understood correctly, this issue was just for the molecule description in the observation window (first bullet). I think this has been updated to match https://docs.google.com/spreadsheets/d/19eCpnjjaJ8it0Tzdbk_159xhH_WJVPVmv_jHwnSdw3s/edit#gid=0. @Matthew-Moore240 can you confirm and review latest?

**Comment by terracoda** [comment]
@jessegreenberg, yes, these are just for the Molecule Descriptions in the Observation Window - their main phases as they update in the back ground. The idea is that if a user pauses the sim while a molecule is in a particular phase and then goes and reads the Observation Window, they will get an accurate state description (an accurate current state).

@Matthew-Moore240, let me know if you need any assistance. For verification of these description, the A11y View will work great.

**Comment by jessegreenberg** [comment]
The dynamic descriptions for the observation window are done, and can be reviewed at https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-dev.46/phet/molecules-and-light_a11y_view.html

**Comment by terracoda** [comment]
@jessegreenberg and @Matthew-Moore240, the molecule phases in the in the Observation all look great.

I found some awkwardness after NO<sub>2</sub> and O<sub>3</sub>) break apart and float away. Once the pieces of the molecules are fully off screen, there's no indication in the Observation Window that they are gone. Though there is an indication in the screen summary.

This situation also causes some awkwardness with the context responses.
![Screen Shot 2020-04-16 at 8 55 39 AM](https://user-images.githubusercontent.com/1902116/79496842-50f86480-8001-11ea-99b7-431377b1e8ab.png)

@Matthew-Moore240, what do you think of indicating the "floating away" phase is over, i.e. the pieces are gone. I am not sure it is necessary that we name the pieces or the molecule. We could use something generic:
- Molecule pieces gone. Reset or change molecule.

This phrase essentially captures the final phase of the break apart, and if ever read together with screen summary, it might support putting together why the , "Ultraviolet light source emits photons directly at empty space. Reset or change molecule."

So basically, for the molecules that break apart there would be 4 phases for the dynamic description instead of 3.

**Nitrogen Dioxide:** 
1. Ultraviolet light source is off and points directly at Nitrogen Dioxide molecule.
2. Ultraviolet photon absorbed, Nitrogen Dioxide molecule breaks into NO and O.
3. NO and O floating away. Reset or change molecule.
4. Molecule pieces gone. Reset or change molecule.

**And for Ozone:**
1. Ultraviolet photon passes through Ozone molecule.
2. Ultraviolet photon absorbed, Ozone molecule breaks into O2 and O.
3. O2 and O floating away. Reset or change molecule.
4. Molecule pieces gone. Reset or change molecule.

@Matthew-Moore240, do you like this approach?
- [ ] yes
- [ ] no

@jessegreenberg, can you tell when the pieces are off screen, or guess when they are gone?
- [ ] yes
- [ ] no

If yes, in both cases we can adjust the content according to @Matthew-Moore240 's wishes and run the content by @arouinfar and @BLFiedler.

**Comment by terracoda** [comment]
We can deal with the awkwardness in context response in the context response issue #285 once I finish reviewing with a screen reader.

**Comment by terracoda** [comment]
@BLFiedler and @arouinfar, we need a wording verification for Phase 4 of the break apart situation noted in  https://github.com/phetsims/molecules-and-light/issues/323#issuecomment-614855745

After hearing the phase 3's for either O<sub>3</sub> or NO<sub>2</sub>, what would be most correct? 
- Molecule pieces gone. Reset or change molecule.
- Molecules gone. Reset or change molecule.
- Molecule gone. Reset or change molecule.

**Comment by terracoda** [comment]
Thanks @arouinfar. I'll assign to @jessegreenberg for the implementation.

@jessegreenberg, please add this Phase 4 for both molecules that break apart:
- Molecule pieces gone. Reset or change molecule.

**Comment by jessegreenberg** [comment]
Over slack I asked if this has an impact on the description for the observation window since the content is shared. @terracoda said that when molecules are removed from the observation window, the description should change from 
(what is there currently)
> NO and O floating away. Reset or change molecule.

to
> Molecule pieces gone. Reset or change molecule.

**Comment by terracoda** [comment]
This last description is like a fourth phase and final phase of the molecules demise.

**Comment by terracoda** [comment]
@jessegreenberg, oh maybe there are only 3 phases in the Observation Window
1. Ultraviolet photon absorbed, Nitrogen Dioxide molecule breaks into NO and O.
2. NO and O floating away. 
3. Molecule pieces gone. Reset or change molecule.

Is this how it would work?

**Comment by jessegreenberg** [comment]
For the description item I was thinking there would only be two phases.
1. Ultraviolet photon absorbed, Nitrogen Dioxide molecule breaks into NO and O.
2. Molecule pieces gone. Reset or change molecule.

**Comment by terracoda** [comment]
Oh, phase 1, "photon passes", doesn't always happen.

**Comment by terracoda** [comment]
For the actual excitation there are only 2 phases, but the sim can be paused at any moment, so it can be frozen in one of the 4 phases, right?

**Comment by jessegreenberg** [comment]
Im just thinking about the break apart case for now, the possible states are
- Light source off.
- Photons passing through.
- Molecule breaks apart.
- Constituent molecules removed from window.

"Ultraviolet photon absorbed, Nitrogen Dioxide molecule breaks into NO and O." and
"NO and O floating away." both describe the third state, so I don't know how/when to show both.

So at the moment the possible descriptions are
> Ultraviolet light source is off and points directly at Nitrogen Dioxide molecule.
> Ultraviolet photon passes through Nitrogen Dioxide molecule.
> Ultraviolet photon absorbed, Nitrogen Dioxide molecule breaks into NO and O.
> Molecule pieces gone. Reset or change molecule.

Is this correct?

**Comment by jessegreenberg** [comment]
Sorry, is that what you meant by 4 phases?

**Comment by terracoda** [comment]
Oh, I wasn't counting the off phase. There were already 4 phases. There will now be 5 phases.

**Comment by jessegreenberg** [comment]
We discussed this issue during design meeting today and decided to make the following changes:

- In the observation window escription item after break apart, add "{{A}} and {{B}} floating away." so it will look like
>"Ultraviolet photon absorbed, Nitrogen Dioxide molecule breaks into NO and O. NO and O floating away."

- To match timing of the "Reset Molecule" button, this description and matching alerts should appear after the *first* constituent molecule is removed rather than the second.

**Comment by jessegreenberg** [comment]
OK, I think all discussed in this issue is complete. Assigning to @terracoda and @Matthew-Moore240 to review.

**Comment by terracoda** [comment]
The Observation Window description looks great, and not related to this issue, I think the Context Responses look good, too.
